bazel/example/WORKSPACE.bazel

```diff
@@ -11,8 +11,8 @@ local_repository(
 
 # http_archive(
 #     name = "com_github_p4lang_p4runtime",
-#     urls = ["https://github.com/p4lang/p4runtime/archive/v1.2.0.tar.gz"],
-#     strip_prefix = "p4runtime-1.2.0/proto",
+#     urls = ["https://github.com/p4lang/p4runtime/archive/v1.3.0.tar.gz"],
+#     strip_prefix = "p4runtime-1.3.0/proto",
```

Comment:
Also update `tag = "v1.2"` to `1.3` below.

Reply:
shouldn't the tag be v1.2.0 / v1.3.0? There is no v1.2 github tag right now

Reply:
Good point!

Reply:
Fixed